### YamlMime:ManagedReference
items:
- uid: Microsoft.VisualBasic.CompilerServices.InternalErrorException
  id: InternalErrorException
  children:
  - Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor
  - Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor(System.String)
  - Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: InternalErrorException
  nameWithType: InternalErrorException
  fullName: Microsoft.VisualBasic.CompilerServices.InternalErrorException
  type: Class
  summary: "内部の Visual Basic コンパイラ エラーに対してスローされる例外。"
  remarks: "このクラスは、Visual Basic コンパイラをサポートしているし、コードから直接使用するものではありません。"
  syntax:
    content: >-
      [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]

      public sealed class InternalErrorException : Exception
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor
  id: '#ctor'
  parent: Microsoft.VisualBasic.CompilerServices.InternalErrorException
  langs:
  - csharp
  name: InternalErrorException()
  nameWithType: InternalErrorException.InternalErrorException()
  fullName: Microsoft.VisualBasic.CompilerServices.InternalErrorException.InternalErrorException()
  type: Constructor
  assemblies:
  - Microsoft.VisualBasic
  namespace: Microsoft.VisualBasic.CompilerServices
  summary: "新しいインスタンスを初期化、 <xref href=&quot;Microsoft.VisualBasic.CompilerServices.InternalErrorException&quot;> </xref>クラスです。"
  remarks: "このクラスは、Visual Basic コンパイラをサポートしているし、コードから直接使用するものではありません。"
  syntax:
    content: public InternalErrorException ();
    parameters: []
  overload: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: Microsoft.VisualBasic.CompilerServices.InternalErrorException
  langs:
  - csharp
  name: InternalErrorException(String)
  nameWithType: InternalErrorException.InternalErrorException(String)
  fullName: Microsoft.VisualBasic.CompilerServices.InternalErrorException.InternalErrorException(String)
  type: Constructor
  assemblies:
  - Microsoft.VisualBasic
  namespace: Microsoft.VisualBasic.CompilerServices
  summary: "新しいインスタンスを初期化、 <xref href=&quot;Microsoft.VisualBasic.CompilerServices.InternalErrorException&quot;> </xref>クラス、エラー メッセージを指定します。"
  remarks: "値、`message`パラメーターがの値として設定されて、<xref:System.Exception.Message%2A>プロパティ</xref:System.Exception.Message%2A>。       このクラスは、Visual Basic コンパイラをサポートしているし、コードから直接使用するものではありません。"
  syntax:
    content: public InternalErrorException (string message);
    parameters:
    - id: message
      type: System.String
      description: "エラーを説明するメッセージ。"
  overload: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: Microsoft.VisualBasic.CompilerServices.InternalErrorException
  langs:
  - csharp
  name: InternalErrorException(String,Exception)
  nameWithType: InternalErrorException.InternalErrorException(String,Exception)
  fullName: Microsoft.VisualBasic.CompilerServices.InternalErrorException.InternalErrorException(String,Exception)
  type: Constructor
  assemblies:
  - Microsoft.VisualBasic
  namespace: Microsoft.VisualBasic.CompilerServices
  summary: "新しいインスタンスを初期化、 <xref href=&quot;Microsoft.VisualBasic.CompilerServices.InternalErrorException&quot;> </xref>クラス、エラー メッセージと内部例外を指定します。"
  remarks: "値、`message`パラメーターがの値として設定されて、<xref:System.Exception.Message%2A>プロパティ</xref:System.Exception.Message%2A>。 値、`innerException`パラメーターがの値として設定されて、<xref:System.Exception.InnerException%2A>プロパティ</xref:System.Exception.InnerException%2A>。       このクラスは、Visual Basic コンパイラをサポートしているし、コードから直接使用するものではありません。"
  syntax:
    content: public InternalErrorException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "エラーを説明するメッセージ。"
    - id: innerException
      type: System.Exception
      description: "現在の内部例外の原因となった例外。"
  overload: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor
  parent: Microsoft.VisualBasic.CompilerServices.InternalErrorException
  isExternal: false
  name: InternalErrorException()
  nameWithType: InternalErrorException.InternalErrorException()
  fullName: Microsoft.VisualBasic.CompilerServices.InternalErrorException.InternalErrorException()
- uid: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor(System.String)
  parent: Microsoft.VisualBasic.CompilerServices.InternalErrorException
  isExternal: false
  name: InternalErrorException(String)
  nameWithType: InternalErrorException.InternalErrorException(String)
  fullName: Microsoft.VisualBasic.CompilerServices.InternalErrorException.InternalErrorException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor(System.String,System.Exception)
  parent: Microsoft.VisualBasic.CompilerServices.InternalErrorException
  isExternal: false
  name: InternalErrorException(String,Exception)
  nameWithType: InternalErrorException.InternalErrorException(String,Exception)
  fullName: Microsoft.VisualBasic.CompilerServices.InternalErrorException.InternalErrorException(String,Exception)
- uid: Microsoft.VisualBasic.CompilerServices.InternalErrorException.#ctor*
  parent: Microsoft.VisualBasic.CompilerServices.InternalErrorException
  isExternal: false
  name: InternalErrorException
  nameWithType: InternalErrorException.InternalErrorException
